examples/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.8.0"
 edition = "2021"
 
 [dependencies]
-odra = { path = "../odra", default-features = false }
+odra = { path = "../odra", version = "0.8.0", default-features = false }
```

Comment:
No need to do this in `examples`, this crate is not being published.

Reply:
ok